Global/CodeComposerStudio.gitignore

```diff
@@ -0,0 +1,16 @@
+# =========================
+# Code Composer Studio Files
+# =========================
```

Comment:
Most of the project templates maintained here don't have a header like this. The file name should be good enough to distinguish.